### ISSUE TITLE
Support cocoapods

### ISSUE THREAD

**Issue by moh-abk** [vyshane/grpc-swift-combine#10]
Due to this issue - https://forums.swift.org/t/swift-packages-in-multiple-targets-results-in-this-will-result-in-duplication-of-library-code-errors/34892/38 - SPM is kind of broken at the moment. It'll be great if this made it to cocoapods 😉

**Comment by vyshane** [comment]
I've just pushed the new podspec.

```
--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  CombineGRPC (0.16.0) successfully published
 📅  August 18th, 22:38
 🌎  https://cocoapods.org/pods/CombineGRPC
 👍  Tell your friends!
--------------------------------------------------------------------------------
```

Can you try to install it by adding the following to your Podfile?

```
pod 'CombineGRPC', '~> 0.16'
```

**Comment by moh-abk** [comment]
Awesome - just pulled it but it's not showing the source files.
Might be related to - https://github.com/vyshane/grpc-swift-combine/blob/master/CombineGRPC.podspec#L19

![image](https://user-images.githubusercontent.com/7813457/90530686-13e9dd80-e16d-11ea-8090-ef8dd8b351de.png)

**Comment by vyshane** [comment]
Yikes. That was my first foray into creating a podspec. Thanks for spotting the issue. I've fixed it. However, it looks like we are blocked by an upstream issue grpc/grpc-swift#928.

**Comment by vyshane** [comment]
Upstream Swift gRPC have fixed their CocoaPod in release `v1.0.0-alpha.19`. I have updated the dependency in CombineGRPC and released `v0.17.0`. You can now install the pod by adding 'CombineGRPC', '~> 0.17' to your Podfile.

**Comment by moh-abk** [comment]
thanks @vyshane -- can confirm it works!
